src/utils/rest-api.js

```diff
@@ -1189,6 +1189,25 @@ export function connectNotificationsWebsocket(studyUuid) {
     return rws;
 }
 
+export function connectDeletedStudyNotificationsWebsocket(studyUuid) {
+    // The websocket API doesn't allow relative urls
+    const wsbase = document.baseURI
+        .replace(/^http:\/\//, 'ws://')
+        .replace(/^https:\/\//, 'wss://');
+    const wsadress =
+        wsbase +
+        PREFIX_DIRECTORY_NOTIFICATION_WS +
```

Comment:
We already have a websocket open to the directory-notification-server, why using another one ?

Reply:
Because I wanted one that received specifically the study deletion so it's easy to read and understand. Websocket send really light messages so I thought this was an ok solution. Maybe it's not, you tell me.

Reply:
Ok will be discuss maybe later